### PR TITLE
[Merged by Bors] - chore(Data/Set/List): nthLe -> get migration

### PR DESCRIPTION
**Pull request by grunweg**
---

Extracted from #12350.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Set/List.lean [hunk 1]
```diff
@@ -36,14 +36,15 @@ theorem range_list_map_coe (s : Set α) : range (map ((↑) : s → α)) = { l |
 #align set.range_list_map_coe Set.range_list_map_coe
 
 @[simp]
-theorem range_list_nthLe : (range fun k : Fin l.length => l.nthLe k k.2) = { x | x ∈ l } := by
+theorem range_list_get : (range fun k : Fin l.length => l.get ⟨k, k.2⟩) = { x | x ∈ l } := by
```
Comment (urkud) [inline Mathlib/Data/Set/List.lean:39]:
```suggestion
theorem range_list_get : range l.get = { x | x ∈ l } := by
```

**Comment by grunweg** [comment]
awaiting-review

**Comment by urkud** [comment]
Thanks! Once CI is happy, please merge this PR using `bors r+`
bors d+

P.S.: I pushed a minor golf.

**Comment by grunweg** [comment]
Thanks for the golf and review.
bors r+

**Pull request closed without merging**